util/chplenv/chpl_gpu.py

```diff
@@ -5,29 +5,52 @@
 import chpl_llvm
 from utils import error, memoize, run_command
 
+# Format:
+#   environment variable
+#   program to locate SDK folder
+#   depth of program within SDK folder
+GPU_TYPES = {
+    "cuda": ("CHPL_CUDA_PATH", "nvcc", 2),
```

Comment:
I think I'd change this to: "nvidia" and "amd" just so both are referring to vendors rather than one referring to a programming model and one referring to a vendor.